students/larry_mburu/Lesson06/test_mailroom.py

```diff
@@ -0,0 +1,98 @@
+import mailroom
+import os
+from operator import itemgetter
+
+
+#test functions for Send Thank You!!
+
+def test_get_donor_list(): 
+    actual  = mailroom.get_donor_list('list')
+    expected = ['William Gates III', 'Mark Zuckerberg', 'Jeff Bezo', 'Paul Allen', 'Bill Gates']
+    assert actual == expected
+
+def test_add_donor_to_list(): 
+    '''
+    test new user not in DB
+    '''
+    mailroom.add_donar_to_list('James Comey')
+    assert mailroom.donations_per_individual['James Comey'] == [100.0]
+
+def test_add_to_donation_database(): 
+    assert mailroom.add_to_donation_database('James Comey', 100) == None
```

Comment:
this doesn't really test anything, right? a function that has not return value can't really be tested by asserting that it returns nothing.